core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFormatterUtil.java

```diff
@@ -50,52 +49,70 @@ interface DateTimeFormatHandler {
-          String.format("'%d'", CalendarLookup.getWeekNumber(0, date.toLocalDate())))
-      .put("%u", (date) -> // %u Week where Monday is the first day - WEEK() mode 1
-          String.format("'%d'", CalendarLookup.getWeekNumber(1, date.toLocalDate())))
-      .put("%V", (date) -> // %V Week where Sunday is the first day - WEEK() mode 2 used with %X
-          String.format("'%d'", CalendarLookup.getWeekNumber(2, date.toLocalDate())))
-      .put("%v", (date) -> // %v Week where Monday is the first day - WEEK() mode 3 used with %x
-          String.format("'%d'", CalendarLookup.getWeekNumber(3, date.toLocalDate())))
-      .put("%X", (date) -> // %X Year for week where Sunday is the first day, 4 digits used with %V
-          String.format("'%d'", CalendarLookup.getYearNumber(2, date.toLocalDate())))
-      .put("%x", (date) -> // %x Year for week where Monday is the first day, 4 digits used with %v
-          String.format("'%d'", CalendarLookup.getYearNumber(3, date.toLocalDate())))
-      .build();
-
-  //Handlers for the time_format function.
-  //Some format specifiers return 0 or null to align with MySQL.
-  //https://dev.mysql.com/doc/refman/8.0/en/date-and-time-functions.html#function_time-format
+          .put("%a", (date) -> "EEE") // %a => EEE - Abbreviated weekday name (Sun..Sat)
+          .put("%b", (date) -> "LLL") // %b => LLL - Abbreviated month name (Jan..Dec)
+          .put("%c", (date) -> "MM") // %c => MM - Month, numeric (0..12)
+          .put("%d", (date) -> "dd") // %d => dd - Day of the month, numeric (00..31)
+          .put("%e", (date) -> "d") // %e => d - Day of the month, numeric (0..31)
+          .put("%H", (date) -> "HH") // %H => HH - (00..23)
+          .put("%h", (date) -> "hh") // %h => hh - (01..12)
+          .put("%I", (date) -> "hh") // %I => hh - (01..12)
+          .put("%i", (date) -> "mm") // %i => mm - Minutes, numeric (00..59)
+          .put("%j", (date) -> "DDD") // %j => DDD - (001..366)
+          .put("%k", (date) -> "H") // %k => H - (0..23)
+          .put("%l", (date) -> "h") // %l => h - (1..12)
+          .put("%p", (date) -> "a") // %p => a - AM or PM
+          .put("%M", (date) -> "LLLL") // %M => LLLL - Month name (January..December)
+          .put("%m", (date) -> "MM") // %m => MM - Month, numeric (00..12)
+          .put("%r", (date) -> "hh:mm:ss a") // %r => hh:mm:ss a - hh:mm:ss followed by AM or PM
+          .put("%S", (date) -> "ss") // %S => ss - Seconds (00..59)
+          .put("%s", (date) -> "ss") // %s => ss - Seconds (00..59)
+          .put("%T", (date) -> "HH:mm:ss") // %T => HH:mm:ss
+          .put("%W", (date) -> "EEEE") // %W => EEEE - Weekday name (Sunday..Saturday)
+          .put("%Y", (date) -> "yyyy") // %Y => yyyy - Year, numeric, 4 digits
+          .put("%y", (date) -> "yy") // %y => yy - Year, numeric, 2 digits
+          // The following are not directly supported by DateTimeFormatter.
+          .put(
```

Comment:
nit: you can move inline comments here up too.

Reply:
Sadly lines are too long to be on one line in spotless :(